amgut/db/patches/0041.sql

```diff
@@ -177,9 +230,10 @@ CREATE INDEX idx_barcode_sequence_plate_plate_type_id ON pm.barcode_sequence_pla
 
 CREATE TABLE pm.barcode_sequence_plate_layout (
     barcode_sequence_plate_id bigint  NOT NULL,
-    col                  smallint  NOT NULL,
-    row                  smallint  NOT NULL,
-    barcode_sequence     varchar  ,
+    col                       smallint  NOT NULL,
+    row                       smallint  NOT NULL,
+    barcode_sequence          varchar  NOT NULL,
```

Comment:
this PR is for target gene, but this table is not specific to target gene. Is this table structure sufficient for WGS given dual indexing strategies being used? If not, should the table name include "target_gene"?